wagtail/admin/views/chooser.py

```diff
@@ -92,6 +94,62 @@ def can_choose_page(
     return True
 
 
+class PageChooserTable(Table):
+    classname = "listing chooser"
+
+    def get_row_classname(self, page):
+        classnames = []
+        if page.is_parent_page:
+            classnames.append("parent-page")
+        if not page.live:
+            classnames.append("unpublished")
+        if not page.can_choose:
+            classnames.append("disabled")
+
+        return " ".join(classnames)
```

Comment:
```suggestion
        return " ".join(classnames)

    def get_context_data(self, parent_context):
        context = super().get_context_data(parent_context)
        context["show_locale_labels"] = parent_context.get("show_locale_labels", False)
        return context
```

otherwise the columns don't get `show_locale_labels` passed down to them

Reply:
Good catch! Done in [dbd9004](https://github.com/wagtail/wagtail/pull/8485/commits/dbd9004586ad6c58a7ffa17dfcb59c5d7a483e36).